etc/packaging/appimages/orb-slam3-module-aarch64.yml

```diff
@@ -0,0 +1,77 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
```

Comment:
[note to self] I was curious what the difference between this recipe and the one generated by `appimage-builder --generate` would be. I was able to produce a recipe:

on macbook:
* start docker desktop application

on macbook, from root of viam-orb-slam3 repo:
* `$ canon`

now from within canon on macbook, from root of viam-orb-slam3 repo:
* `$ sudo pip3 install appimage-builder` (though it seemed like it already existed)
* follow viam-orb-slam3 [build instructions](https://github.com/viamrobotics/viam-orb-slam3#setup-build-and-run-the-binary)
* `$ mkdir AppDir`
* `$ cp ./viam-orb-slam3/bin/orb_grpc_server AppDir/`
* `$ appimage-builder --generate`

This prompted me to answer some questions. With these answers:

```
$ appimage-builder --generate
INFO:Generator:Searching AppDir
? ID [Eg: com.example.app]: com.viam.orb-slam3-module
? Application Name: orb-slam3-module
? Icon: viam-server
? Executable path relative to AppDir [usr/bin/app]: dontknow
? Arguments [Default: $@]: $@
? Version [Eg: 1.0.0]: latest
? Update Information [Default: guess]: guess
? Architecture: x86_64
INFO:AppRuntimeAnalyser:/usr/bin/strace -f -E LD_LIBRARY_PATH= -e trace=openat --status=successful /host/AppDir/dontknow $@
WARNING:AppRuntimeAnalyser:/usr/bin/strace -f -E LD_LIBRARY_PATH= -e trace=openat --status=successful /host/AppDir/dontknow $@ exited with code 1
WARNING:AppRuntimeAnalyser:This may produce an incomplete/wrong recipe. Please make sure that the application runs properly.
INFO:AppRuntimeAnalyser:Reading PT_INTERP from executables
INFO:FilePackageResolver:/usr/bin/dpkg-query -S
dpkg-query: error: --search needs at least one file name pattern argument

Use --help for help about querying packages.
INFO:dpkg-query:/usr/bin/dpkg-query -W -f=${binary:Package}: ${Depends}\n
INFO:Generator:Recipe generation completed.
```

it produced this `AppImageBuilder.yml` at the root of the repo:

```
eshamaharishi@Eshas-MacBook-Air viam-orb-slam3 % cat AppImageBuilder.yml
# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
version: 1
AppDir:
  path: /host/AppDir
  app_info:
    id: com.viam.orb-slam3-module
    name
    icon: viam-server
    version: latest
    exec: dontknow
    exec_args: $@
  apt:
    arch: []
    allow_unauthenticated: true
    sources:
    - sourceline: deb http://deb.debian.org/debian bullseye main
    - sourceline: deb http://deb.debian.org/debian-security bullseye-security main
    - sourceline: deb http://deb.debian.org/debian bullseye-updates main
    - sourceline: deb [signed-by=/usr/share/keyrings/raspberrypi.gpg] http://archive.raspberrypi.org/debian/
        bullseye main
    - sourceline: deb [signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian
        bullseye stable
    - sourceline: deb [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x
        bullseye main
    - sourceline: deb [signed-by=/usr/share/keyrings/viam-google.gpg] https://us-apt.pkg.dev/projects/static-file-server-310021
        bullseye main
    - sourceline: deb http://deb.debian.org/debian bullseye-backports main
    include: []
  files:
    include: []
    exclude:
    - usr/share/man
    - usr/share/doc/*/README.*
    - usr/share/doc/*/changelog.*
    - usr/share/doc/*/NEWS.*
    - usr/share/doc/*/TODO.*
  test:
    fedora-30:
      image: appimagecrafters/tests-env:fedora-30
      command: ./AppRun
    debian-stable:
      image: appimagecrafters/tests-env:debian-stable
      command: ./AppRun
    archlinux-latest:
      image: appimagecrafters/tests-env:archlinux-latest
      command: ./AppRun
    centos-7:
      image: appimagecrafters/tests-env:centos-7
      command: ./AppRun
    ubuntu-xenial:
      image: appimagecrafters/tests-env:ubuntu-xenial
      command: ./AppRun
AppImage:
  arch: aarch64
  update-information: guess
```

If I try to use this recipe, it fails, I think because the AppDir -> apt -> arch array is empty:

```
$ appimage-builder --recipe AppImageBuilder.yml
INFO:main:Running main script
INFO:main:Running apt deploy
Traceback (most recent call last):
  File "/usr/local/bin/appimage-builder", line 8, in <module>
    sys.exit(__main__())
  File "/usr/local/lib/python3.9/dist-packages/appimagebuilder/__main__.py", line 50, in __main__
    invoker.execute(commands)
  File "/usr/local/lib/python3.9/dist-packages/appimagebuilder/invoker.py", line 29, in execute
    command()
  File "/usr/local/lib/python3.9/dist-packages/appimagebuilder/commands/apt_deploy.py", line 42, in __call__
    apt_venv = self._setup_apt_venv()
  File "/usr/local/lib/python3.9/dist-packages/appimagebuilder/commands/apt_deploy.py", line 60, in _setup_apt_venv
    apt_venv = Venv(
  File "/usr/local/lib/python3.9/dist-packages/appimagebuilder/modules/deploy/apt/venv.py", line 47, in __init__
    self._write_apt_conf(user_options, architectures)
  File "/usr/local/lib/python3.9/dist-packages/appimagebuilder/modules/deploy/apt/venv.py", line 87, in _write_apt_conf
    "APT::Architecture": architectures[0],
IndexError: list index out of range
```